spec/features/user_can_add_items_to_cart_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.feature "visitor can add item to cart and then see it in the cart" do
   scenario "visitor can add item to cart from item index" do
 
-    item_1 = Item.create(title: "Paperclip", description: "This is an artisinal paperclip!", price: 5000, image_url: "http://tectonicablog.com/wp-content/uploads/2013/01/023-455x606.jpg")
+    item = Fabricate(:item)
```

Comment:
This is such a HUGE improvement for our tests. Thanks for getting Fabrication up and running!